product-scenarios/2-Bridging-systems-that-communicate-in-different-protocols/2.5-Switching-from-UDP-to-HTTP-or-HTTPS/02-synapseConfig/artifact.xml

```diff
@@ -1,35 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?><artifacts>
```

Comment:
I think we do not need this artifact.xml file. We maintain the synapse project in scenario level. Shall we remove this file?